Makefile.maven

```diff
@@ -10,7 +10,7 @@ java_install_root:
 	echo "Installing root pom ..."
 	mvn $(MVN_ARGS) install -f ../pom.xml -N
 
-java_install: java_install_root
+java_install:
```

Comment:
This is important when you use `make all` only in one of the submodules. I think we should keep it as it is.